src/lib/test/wrapper.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { Provider } from 'react-redux';
+import { Router } from 'react-router';
+import { createStore } from 'redux';
+import { createBrowserHistory } from 'history';
+
+import { rootReducer } from 'src/modules';
+import { languageMap } from 'src/translations';
+
+const browserHistory = createBrowserHistory();
+const store = createStore(rootReducer);
+
+const locale = 'en';
+
+export const TestComponentWrapper: React.FC = ({ children }) => {
```

Comment:
Please, use "App" for this functional, not "TestComponentWrapper"

Reply:
Unit Tests will be handled more correct here - #380

Reply:
This is wrapper for a component which is under the unit test. This is not the application's wrapper. Are you sure to rename it to `App`?